web/locales/common-voice/en/pages/contribute/write.ftl

```diff
@@ -151,11 +151,13 @@ write-contribute-menu-tooltip = Add and Review Sentences, Add Questions, Transcr
 add-sentences-menu-item-tooltip = Add sentences in your language
 review-sentences-menu-item-tooltip = Review sentences in your language
 add-questions-menu-item-tooltip = Add questions in your language
+review-questions-menu-item-tooltip = Review added questions in your language
```

Comment:
We can just say "Review questions in your language" ... the "added" is redundant.